src/main/kotlin/org/rust/lang/utils/CharEx.kt

```diff
@@ -0,0 +1,6 @@
+package org.rust.lang.utils
+
+/**
+ * Determines if the char is a Rust whitespace character.
+ */
+fun Char.isRustWhitespaceChar(): Boolean = equals(' ') || equals('\r') || equals('\n') || equals('\t')
```

Comment:
Let's move it to `RustEscapesLexer`. Util modules are prone to uncontrollable growth, so it's better to avoid them if possible.